src/main/scala/cromiam/webservice/CromIamApiService.scala

```diff
@@ -54,6 +55,21 @@ trait CromIamApiService extends Directives with SprayJsonSupport with DefaultJso
     Http().singleRequest(cromwellRequest)
   }
 
+  /**
```

Comment:
Wait, I'm confused about this. We document that version and stats have `api`:

https://github.com/broadinstitute/cromwell#get-apiengineversionversion